src/Compilers/CSharp/Portable/Binder/Binder_TupleOperators.cs

```diff
@@ -55,6 +96,13 @@ internal partial class Binder
                 return BindTupleBinaryOperatorNestedInfo(node, kind, left, right, diagnostics);
             }
 
+            bool leftNull = left.IsLiteralNull();
+            bool rightNull = right.IsLiteralNull();
+            if (leftNull && rightNull)
+            {
+                return new TupleBinaryOperatorInfo.NullNull(kind);
+            }
```

Comment:
Are we planning on using `leftNull` and `rightNull` later in the method? If not, consider inlining so we only check `right.IsLiteralNull()` if necessary.